proto/cosmwasm/wasm/v1/types.proto

```diff
@@ -37,7 +41,11 @@ message AccessTypeParam {
 message AccessConfig {
   option (gogoproto.goproto_stringer) = true;
   AccessType permission = 1 [ (gogoproto.moretags) = "yaml:\"permission\"" ];
+
+  // Address
+  // Deprecated: replaced by addresses
```

Comment:
The decision to deprecate is to not maintain two fields here?
Maybe worth it 🤷

Reply:
It feels wasted to have both fields and may be confusing when used in the API. Added https://github.com/CosmWasm/wasmd/issues/986 to follow up on the deprecation tag